conference.js

```diff
@@ -3082,11 +3083,11 @@ export default {
      * @param {boolean} doDisconnect - Whether leaving the room should also terminate the connection.
      * @returns {Promise}
      */
-    async leaveRoom(doDisconnect = true) {
+    async leaveRoom(reason, doDisconnect = true) {
```

Comment:
Please add the new parameter at the end, and give it some default value like the empty string instead of making it mandatory.